apps/admin-x-settings/src/MainContent.tsx

```diff
@@ -36,15 +33,7 @@ const MainContent: React.FC = () => {
         const handleKeyDown = (event: KeyboardEvent) => {
             if (event.key === 'Escape') {
                 confirmIfDirty(isDirty, () => {
-                    if (config.labs.ui60) {
-                        navigateAway('/analytics');
-                    } else {
-                        if (hasActivityPub && hasAdminAccess(currentUser)) {
-                            navigateAway('/activitypub');
-                        } else {
-                            navigateAway('/dashboard');
-                        }
-                    }
+                    navigateAway('/analytics');
```

Comment:
I think we can now change this to `navigateAway('/')` instead. That way, Ghost Admin renders the default sidebar menu (which should be analytics in 6.x for most users, but might be something else if traffic analytics is not available for some reason, e.g. to self-hosters)